src/main/java/com/nvidia/spark/rapids/jni/GpuTimeZoneDB.java

```diff
@@ -198,16 +247,27 @@ private void doLoadData() {
             if (zoneRules.isFixedOffset()) {
               data.add(
                   new HostColumnVector.StructData(Long.MIN_VALUE, Long.MIN_VALUE,
-                      zoneRules.getOffset(Instant.now()).getTotalSeconds())
+                      zoneRules.getOffset(Instant.now()).getTotalSeconds(), Long.MIN_VALUE)
               );
             } else {
               // Capture the first official offset (before any transition) using Long min
               ZoneOffsetTransition first = transitions.get(0);
               data.add(
                   new HostColumnVector.StructData(Long.MIN_VALUE, Long.MIN_VALUE,
-                      first.getOffsetBefore().getTotalSeconds())
+                      first.getOffsetBefore().getTotalSeconds(), Long.MIN_VALUE)
               );
               transitions.forEach(t -> {
+                // A simple approach to transform LocalDateTime to a value which is proportional to
```

Comment:
Again this is kind of confusing and it would be nice to have a reference to where this idea came from, or some more comments explaining how/why this works.

Reply:
Will remove this.
Refer to https://github.com/NVIDIA/spark-rapids-jni/pull/1539#discussion_r1461590178